src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.Converters.cs

```diff
@@ -30,8 +30,8 @@ public sealed partial class JsonSerializerOptions
         [RequiresUnreferencedCode(JsonSerializer.SerializationUnreferencedCodeMessage)]
         private void RootBuiltInConverters()
         {
-            s_defaultSimpleConverters ??= GetDefaultSimpleConverters();
```

Comment:
I'd still keep these; both methods incur a whole lot of allocations so would be nice if we could avoid some of these when contending threads attempt to root them.

Reply:
Note is is guarded by an `if` statement previously, so the window of contention is still very small.

Reply:
This will be called every time when calling `GetConverter`, which is public api. It's now somehow confusing to me that about how to use `GetConverter`.